### PR TITLE
ZENKO-3716: add release process

### PR DESCRIPTION
**Pull request by alexanderchan-scality**
Add release instructions

### CODE REVIEW

File: docs/release.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+## Release Process
+
+To release the Zenko and Zenko-base ISOs:
+
+1. Tag and release the Zenko version using the [GitHub release page](https://github.com/scality/Zenko/releases/new).
+1. Retrieve the eve `artifacts_name` of the tagged build. This can be found
+   under the `Build Properties` of the selected eve build.
+1. Tag the selected artifacts using the eve `promote` command:
+   * The `promote` command can be found on the eve builders page.
```
Comment (miniscruff) [inline docs/release.md]: Is this a static link? Or is it unique per build? Finding stuff on eve can be hard so using the URL might help in the future.

File: docs/release.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+## Release Process
+
+To release the Zenko and Zenko-base ISOs:
+
+1. Tag and release the Zenko version using the [GitHub release page](https://github.com/scality/Zenko/releases/new).
+1. Retrieve the eve `artifacts_name` of the tagged build. This can be found
+   under the `Build Properties` of the selected eve build.
+1. Tag the selected artifacts using the eve `promote` command:
+   * The `promote` command can be found on the [eve builders page](https://eve.devsca.com/github/scality/zenko/#/builders/4).
```
Comment (miniscruff) [inline docs/release.md]:
```suggestion
   * The `promote` command can be found on the [eve builders page](https://eve.devsca.com/github/scality/zenko/#/builders/bootstrap).
```

@tcarmet suggests using this URL ( I was curious if 4 was static or not )

**Review by tcarmet — APPROVED** [review]
With @miniscruff suggestion, LGTM

**Review by anurag4DSB — APPROVED** [review]
Apart from Ronnie's comment, looks good.
Thank you so much for this one.

**Pull request merged**